packages/harness/appConfigs/harness/renative.json

```diff
@@ -18,6 +18,9 @@
             "canary": {
                 "id": "io.flexn.harness.alpha",
                 "title": "Harness Alpha"
+            },
+            "production": {
```

Comment:
@pauliusguzas can we remove production from harness at all?

Reply:
@aurimasmi sure